js/calculator.js

```diff
@@ -1,155 +1,267 @@
+            break;
+        case 'squareRoot': //square root
+            screenValue.value = Math.sqrt(screenValue.value);
+            break;
+        case 'square': //find square
+            screenValue.value = Math.pow(screenValue.value,2);
+            break;
+        case 'oneByX': //find 1/x
+            screenValue.value = 1 / screenValue.value;
+            break;
+        case 'abs': //find |x|
+            screenValue.value = Math.abs(screenValue.value);
+            break;
+        case 'exp': //find exp
+            screenValue.value = Math.exp(screenValue.value);
+            break;
+        case 'FE': //find F-E
+            if(screenValue.value == " ")
+            {
+                screenValue.value = '0.e+0';
+            }
+            else{
+                screenValue.value = screenValue.value + ".e+0";
+            }
+            break;
     }
 }
-//three function
-const three = () =>{
-    if(document.querySelector("#display"). value == " "){
-        document.querySelector("#display").value = "3";
-    }
-    else{
-        document.querySelector("#display").value = document.querySelector("#display").value + "3";
+//factorial find
+function factorial(){
+    let fact = 1;
+    for(let i = 1; i <= screenValue.value; i++){
+      fact *= i;
     }
+    screenValue.value = fact;
 }
-//four function
-const four = () =>{
-    if(document.querySelector("#display"). value == " "){
-        document.querySelector("#display").value = "4";
-    }
-    else{
-        document.querySelector("#display").value = document.querySelector("#display").value + "4";
+//X power Y
+function xPowY(){
+    let a = screenValue.value[screenValue.value.length-1];
+    if(a.match(/[0-9]/)){
+        let b = screenValue.value.split("^");
+        screenValue.value = Math.pow(b[0],b[1]);
     }
 }
-//five function
-const five = () =>{
-    if(document.querySelector("#display"). value == " "){
-        document.querySelector("#display").value = "5";
-    }
-    else{
-        document.querySelector("#display").value = document.querySelector("#display").value + "5";
+//Trigonometry function
+function trigonometry(num){
+    let n;
+    switch(num){
+        case 'sin':
+            n = Math.sin(screenValue.value * Math.PI /180)
+            screenValue.value = n;
+            break;
+        case 'cos':
+            n = Math.cos(screenValue.value * Math.PI /180)
+            screenValue.value = n;
+            break;
+        case 'tan':
+            n = Math.tan(screenValue.value * Math.PI /180)
+            screenValue.value = n;
+            break;
+        case 'sinh':
+            n = Math.sinh(screenValue.value * Math.PI /180)
+            screenValue.value = n;
+            break;
+        case 'cosh':
+            n = Math.cosh(screenValue.value * Math.PI /180)
+            screenValue.value = n;
+            break;
+        case 'tanh':
+            n = Math.tanh(screenValue.value * Math.PI /180)
```

Comment:
Add in separated variable and then use it. 
var mathPI = Math.PI /180 ;

Reply:
ok sir